src/components/PaymentRequestButtonElement.test.js

```diff
@@ -74,6 +79,9 @@ describe('PaymentRequestButtonElement', () => {
     expect(elementMock.on.mock.calls[0][0]).toBe('ready');
     expect(onReadyMock).toHaveBeenCalled();
```

Comment:
And if so, add an assertion here that `onReady` received the instance?

Reply:
I wonder if this is something we should just use the types to enforce, (i.e., type `onReady` as `(StripeElement) => void` instead of `Function`).

I'll add a test regardless.